tests/settings.py

```diff
@@ -28,3 +28,5 @@
 AWS_SECRET_ACCESS_KEY = "AWS_SECRET_ACCESS_KEY"
 
 MAX_FILE_UPLOAD_SIZE = 10 * 1024 * 1024  # 10 MB
+
+ACCESS_CONTROL_TYPES = ['private', 'public-read', 'public-read-write', 'authenticated-read']
```

Comment:
Are these types available as constants from anywhere rather than magic strings? if they are I would swap them out throughout the PR for constants but if not maybe its worth creating them so they can be imported by consumers of this library? :)

Reply:
Cool - added as consts